app/components/global-navigation/search.js

```diff
@@ -69,36 +69,49 @@ export default Component.extend({
 
 	requestSuggestionsFromAPI() {
 		const { query } = this.state;
-		const uri = `${this.get('model.suggestions.url')}&query=${query}`;
-
+		const scope = this.get('searchScope');
```

Comment:
Let's add default value for scope e.g. `const scope = this.get('searchScope') || '';`
I think that without it we could end up with requests with param `&scope=undefined`

Reply:
I set default fallback to this-wiki search